src/jit/compphases.h

```diff
@@ -22,6 +22,8 @@
 //     true.
 
 // clang-format off
+//                 enumName                      stringName                        shortName hasChildren measureIR
+//                                                                                                   parent
```

Comment:
I find it easier to have an aligned header than to count the fields and map them back.